scripts/activate-protocol.sh

```diff
@@ -23,12 +23,13 @@ usage() {
     echo "  --tezos-client <filepath>. Path for patched tezos-client executable"
     echo "  --parameters <filepath>. Path to JSON file with protocol parameters."
     echo "  [--fitness <int>]. Protocol activation fitness, default value is $fitness."
+    echo "  [--base-chain <babylonnet | carthagenet>]. Define base chain for your private"
+    echo "    blockchain. Default is 'babylonnet'."
```

Comment:
Since babylonnet will likely die in a week, I think it makes more sense to use carthagennet by default.